lean4/src/putnam_1968_a3.lean

```diff
@@ -1,12 +1,12 @@
 import Mathlib
 open BigOperators
 
-open Finset
+open Finset symmDiff
 
 theorem putnam_1968_a3
-{a : Type}
-(S : Finset a)
-(ha : SDiff (Finset a))
-: ∃ l : List (Finset a), (∀ i : Fin l.length, l[i] ⊆ S) ∧ l[0]! = ∅ ∧ (∀ s ⊆ S, ∃! i : Fin l.length, l[i] = s) ∧
-∀ i ∈ Finset.range (l.length - 1), (l[i]! ⊆ l[i+1]! ∧ (l[i+1]! \ l[i]!).card = 1) ∨ (l[i+1]! ⊆ l[i]! ∧ (l[i]! \ l[i+1]!).card = 1) :=
-sorry
+    (α : Type*) [Finite α] :
+    ∃ (n : ℕ) (s : Fin (2 ^ n) → Set α),
+      s 0 = ∅ ∧
+      (∀ t, ∃! i, s i = t) ∧
+      (∀ i, i + 1 < n → (s i ∆ s (i + 1)).ncard = 1) := by
```

Comment:
Think this should be `i+1 < 2^n`, right?

Reply:
Yes, yikes. Thanks!